api/src/main/resources/workbench-api.yaml

```diff
@@ -7280,3 +7305,93 @@ definitions:
       conceptName:
         type: string
+  AuditLogEntriesResponse:
+    type: object
+    description: >
+      Represents the results of a query into the audit dataset in BigQery without
+      filtering or transformation. This response type is a stopgap until we develop
+      high-level queries with a better semantic mapping to the concepts and rules
+      governing them, which is why a freeform queryMetadata field is tolerable for
+      now.
+    required:
+    - logEntries
+    - queryMetadata
+    properties:
+      logEntries:
+        type: array
+        items:
+          "$ref": "#/definitions/AuditLogEntry"
+      queryMetadata:
+        description: >
+          Since the audit system uses SQL database IDs and the rest of hte API layer
+          does not, it's helpful to return additional properties in order to assist
+          the client in interpreting the results.
+        type: object
+        additionalProperties: true
+  AuditLogEntry:
+    description: >
+      Representation of the BigQuery Audit Log entry, post-translation from Stackdriver logging.
+      All fields are treated as optional here, as there are no non-null constraints in the BigQery
+      dataset.
+
+      It's possible that due to some error now or in the past that some columns might not match
+      their expected values, though we take pains to avoid that. Regardless, nothing in this type
+      definition is stricter than the BigQuery schema.
+    type: object
+    properties:
+      actionId:
+        description: Unique ID for this action. Used for grouping similar events under a
+          single action, though in this representation we're returning a straight list of rows.
+        type: string
+        format: uuid
+      actionType:
+        description: >
+          Defines what action is taking place in this event. (There can be several ACTION_TYPEs
+          in a single Action assigned to different events. String representation of the ActionType
+          enum (not generated).
+        type: string
+      agentId:
+        description: >
+          ID for the Agent (typically a User). If there's a SQL table for this agent type,
+          this this is the PK for that table (e.g. user_id).
+        type: integer
+        format: int64
+      agentUsername:
+        description: >
+          Username of the user or administrator taking the action. This is the agent_email column
+          in BigQuery.
+        type: string
+        format: email
+      agentType:
+        description: >
+          A member of the org.pmiops.workbench.actionaudit.AgentType enum (not generated). Since
+          this is typed as a string in BigQuery (and not constrained to that type), we don't try
+          to narrow the type in the admin API.
+        type: string
+      eventTime:
+        description: >
+          Timestamp recorded at time of the event. Called timestamp in the BigQuery table.
+        type: string
+        format: date-time
```

Comment:
This is the first attempt at the more generic date & time handling. Maps to Joda `DateTime` in Java and Date in TypeScript.